packages/rancher-istio/charts/Chart.yaml

```diff
@@ -9,3 +9,5 @@ annotations:
   catalog.cattle.io/namespace: istio-system
   catalog.cattle.io/release-name: rancher-istio
   catalog.cattle.io/ui-component: istio
+  catalog.cattle.io/requires-gvr: prometheuses.monitoring.coreos.com/v1
+  catalog.cattle.io/auto-install-gvr: monitoringdashboards.monitoring.kiali.io/v1alpha1
```

Comment:
At the moment, this will be overridden based on https://github.com/rancher/charts/blob/531420dcf5ed3089d3dbca6f312304cf853b9b6d/scripts/prepare-crds#L67-L71

While making the changes to the `prepare-crd` script, can you add some logic to make sure we don't override the values if they are already declared in the Chart.yaml? e.g.
```bash
if ! [[ -z ${providesGVR} ]]; then
        if [[ -z $(yq r ${f}/charts/Chart.yaml "annotations[catalog.cattle.io/auto-install-gvr]") ]]; then
	    yq w -i ${f}/charts/Chart.yaml "annotations[catalog.cattle.io/auto-install-gvr]" "${providesGVR}"
        fi
	yq w -i ${f}/charts-crd/Chart.yaml "annotations[catalog.cattle.io/provides-gvr]" "${providesGVR}"
	yq w -i ${f}/charts-crd/Chart.yaml "annotations[catalog.cattle.io/hidden]" "true"
fi
```

Reply:
added below